modules/sdk/node-build/esbuild/getImportBridgePath.mjs

```diff
@@ -0,0 +1,6 @@
+import getFlatName from '../util/getFlatName.mjs';
+
+export default function getImportBridgePath(moduleName) {			
```

Comment:
Can we also put all "bridge" related things in it's own directory? Coming from npm-scripts, I get confused which things are specific for bridge and which isnt.

Also... what exactly are we "bridging"? Is this different than our AMD bridges?

Reply:
This is a different bridge. I call all of them bridges because they are exactly the same and it would probably be worse if we invented fancy names to distinguish them :roll_eyes: .

In this case the files contain code to help esbuild infer what symbols are exported by a given external module.